docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -415,3 +421,31 @@ After logging out of `tsh`, you can log back in without specifying
 
 (!docs/pages/includes/sso/loginerrortroubleshooting.mdx!)
 
+### Teams Not Mapping to Roles
+
+If you are not seeing teams mapping to roles confirm you 
+are using the slug value of the organizations and teams in the connector.
+The GitHub Web application URIs and GitHub API can provide the slug values.
+
+Example with team name `My Team`:
+```yaml
+  teams_to_roles:
+    - organization: ORG-NAME
+      roles:
+        - access
+      team: My Team
```

Comment:
This is invalid, why would we even show it in the docs? Someone is bound to treat this as an example.

Reply:
removed